### PR TITLE
Refactor ckBTCTransferTokens use icrcTransferTokens

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

We can refactor `ckBTCTransferTokens` the other way around that way it still remove duplication and does not introduce too many changes in the code.

# Changes

- use `icrcTransferTokens` in `ckBTCTransferTokens`

### CODE REVIEW

File: frontend/src/lib/services/icrc-accounts.services.ts [hunk 1]
```diff
@@ -32,13 +32,15 @@ export const getIcrcAccountIdentity = (_: Account): Promise<Identity> => {
...
 
-export const loadIcrcToken = ({
-  ledgerCanisterId,
-  certified,
-}: {
-  ledgerCanisterId: Principal;
-  certified: boolean;
-}) => {
+export const loadIcrcToken = (
+  {
```
Comment (lmuntaner) [inline frontend/src/lib/services/icrc-accounts.services.ts:36]: Why do you have these changes in indentation?

It seems an issue with linter?

**Review by lmuntaner — COMMENTED** [review]
Does it make sense to work on this when we haven't finished the functionality for ckETH?

File: frontend/src/lib/services/icrc-accounts.services.ts (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/services/icrc-accounts.services.ts]: I've run the command line `npm run format`.

Comment (peterpeterparker) [reply frontend/src/lib/services/icrc-accounts.services.ts]: Webstorm sh**. I've run the command manually, it's good now.

**Comment by peterpeterparker** [comment]
> Does it make sense to work on this when we haven't finished the functionality for ckETH?

Yes when I'm blocked because I'm waiting on the pipeline that take ages to merge PRs and thus to avoid merge conflicts.

**Comment by peterpeterparker** [comment]
Won't be proceeeded further.

**Pull request closed without merging**